.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,151 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+
+name: Run E2E Tests
```

Comment:
Why can't we have the e2e-build.yml separately so we can reuse it

Reply:
We run into an issue where we call too many nested workflows, so we had to integrate them together.